MojioSDK/Models/Vehicles/Vehicle.swift

```diff
@@ -16,6 +16,18 @@
 import Foundation
 import MojioCore
 
+public enum VehicleType: String, Codable {
+    case physical = "Physical"
+    case ghost = "Ghost"
+    case bluetooth = "Bluetooth"
+    case unknown = "Unknown"
+
+    public init(from decoder: Decoder) throws {
+        let label = try decoder.singleValueContainer().decode(String.self)
```

Comment:
Maybe `value` or `stringValue` is better?

Reply:
@andriimakukha  Thanks. It renamed to rawValue.